build.gradle

```diff
@@ -24,14 +25,20 @@ ext {
     VERSION_NAME = "0.8.1-SNAPSHOT"
     VERSION_CODE = 6
     MIN_SDK_VERSION = 14
-    TARGET_SDK_VERSION = 25
-    COMPILE_SDK_VERSION = 25
-    BUILD_TOOLS_VERSION = '25.0.2'
+    // If you change the SDK version don't forgot to update .travis.yml
+    TARGET_SDK_VERSION = 26
+    COMPILE_SDK_VERSION = 26
+    BUILD_TOOLS_VERSION = '26.0.1'
 
-    supportLibraryVersion = '25.2.0'
+    supportLibraryVersion = '26.0.0'
```

Comment:
why not `26.1.0`?

Reply:
Because of https://github.com/grandcentrix/ThirtyInch/pull/115#discussion_r141189740 🙃

Reply:
Good point!